policyengine_us/variables/gov/states/ar/tax/income/credits/ar_additional_credit.py

```diff
@@ -0,0 +1,31 @@
+from policyengine_us.model_api import *
+
+
+class ar_additional_credit(Variable):
+    value_type = float
+    entity = Person
+    label = "Arkansas additional credit"
+    reference = "https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2023_AR1000F_and_AR1000NR_Instructions.pdf#page=26"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.AR
+
+    def formula(person, period, parameter):
+        # Use individual or joint income based on filing status
+        filing_separately = person.tax_unit("ar_files_separately", period)
+        taxable_income_indiv = person("ar_taxable_income_indiv", period)
+        taxable_income_joint = person("ar_taxable_income_joint", period)
+        taxable_income = where(
```

Comment:
let's make this a new `ar_taxable_income` variable. Also use it in `ar_inflation_relief_credit_person` (and fix the typo on line 18 there)